buddyboss/buddyboss-platform
# Feature profile completion

### All Submissions:






### Changes proposed in this Pull Request:



Worked on the Profile Completion Module based on the instructions, Mockup, and Video provided by Michael in the issue shared here: https://github.com/buddyboss/buddyboss-platform/issues/326

**Closes #326.**

### How to test the changes in this Pull Request:

1. Go to wp-admin > Appearance > Widgets > TEST "(BB) Profile Completion" widget
2.Make sure to change settings in Profile, Widget Form, Profile Avatar, Cover Avatar to make sure progress works as expected.
3. Please go through the **detailed video** shared which will help you TEST work.

### Connected Issue and PR in Theme
https://github.com/buddyboss/buddyboss-theme/issues/344
https://github.com/buddyboss/buddyboss-theme/pull/346


### Proof Screenshots or Video


**13 Minutes long video explaining the end to end functionality:**
https://www.loom.com/share/942d0632aa2f48d2b9a12b44f0b88432

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

- Created Profile Completion Widget.
